From b5ca64c7398d516e9845d849e8306df7ac024bc4 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 11 Apr 2017 13:59:34 -0700
Subject: [PATCH] include asm/ioctl.h explicitly

Needed for _IOC* defines

helps compiling with musl where this file is
not included indirectly

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 server/drivers/hd44780-pifacecad.c | 1 +
 server/drivers/hd44780-spi.c       | 1 +
 2 files changed, 2 insertions(+)

diff --git a/server/drivers/hd44780-pifacecad.c b/server/drivers/hd44780-pifacecad.c
index 43db19c..e2798ad 100644
--- a/server/drivers/hd44780-pifacecad.c
+++ b/server/drivers/hd44780-pifacecad.c
@@ -54,6 +54,7 @@
 #include <unistd.h>
 #include <fcntl.h>
 #include <sys/ioctl.h>
+#include <asm/ioctl.h>
 #include <stdint.h>
 #include <linux/spi/spidev.h>
 
diff --git a/server/drivers/hd44780-spi.c b/server/drivers/hd44780-spi.c
index 5f94333..8a4015b 100644
--- a/server/drivers/hd44780-spi.c
+++ b/server/drivers/hd44780-spi.c
@@ -29,6 +29,7 @@
 #include <unistd.h>
 #include <fcntl.h>
 #include <sys/ioctl.h>
+#include <asm/ioctl.h>
 #include <stdint.h>
 #include <linux/spi/spidev.h>
 
-- 
2.12.2

